src/Directory.Build.props

```diff
@@ -1,4 +1,14 @@
 <Project>
+  <PropertyGroup>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageProjectUrl>http://avaloniaui.net</PackageProjectUrl>
+    <PackageIconUrl>https://avatars2.githubusercontent.com/u/14075148?s=200</PackageIconUrl>
+    <PackageDescription>A multi-platform .NET UI framework. Avalonia supports Windows, Linux and OSX with experimental mobile support for Android and iOS. Avalonia uses a XAML dialect that should feel immediately familiar to anyone coming from WPF, UWP and Xamarin Forms.</PackageDescription>
```

Comment:
may be can copy from readme

Avalonia is a WPF/UWP-inspired cross-platform XAML-based UI framework providing a flexible styling system and supporting a wide range of Operating Systems such as Windows (.NET Framework, .NET Core), Linux (via Xorg), MacOS and with experimental support for Android and iOS.